src/org/robockets/stronghold/robot/shootingwheel/RPMAlign.java

```diff
@@ -1,106 +0,0 @@
-package org.robockets.stronghold.robot.shootingwheel;
```

Comment:
I'm pretty sure. We are not changing rpm speed anymore, so why bother having this bit around? We can always add it back.